Mathlib/Algebra/Order/Floor.lean

```diff
@@ -1395,6 +1396,14 @@ theorem abs_sub_round_div_natCast_eq {m n : ℕ} :
   rw [abs_sub_round_eq_min, Nat.cast_min, ← min_div_div_right hn'.le,
     fract_div_natCast_eq_div_natCast_mod, Nat.cast_sub (m.mod_lt hn).le, sub_div, div_self hn'.ne']
 
+theorem round_half_bot (x : α) : x - 1 / 2 < round x := by
```

Comment:
Please use descriptive names follow the [naming convention](https://leanprover-community.github.io/contribute/naming.html)
```suggestion
theorem sub_half_lt_round (x : α) : x - 1 / 2 < round x := by
```
`bot` refers to `⊥` and there is no `⊥` in your theorem statement